### PR TITLE
Redshift dynamic data masking

### PR DESCRIPTION
**Pull request by clrcrl**
Related to Dave's Discourse post

### CODE REVIEW

File: dynamic-data-masking-redshift/macros/apply_data_masking.sql [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{% set model_cols = adapter.get_columns_in_relation(this) %}
+
+-- create schema if not exists {{ schema }};
+
+create view {{ schema }}.{{ this.identifier }} as (
+
+    select
+        {% for col in model_cols %}
+            {{ log(col.name, info=True) }}
+            {{ col.name }}
```
Comment (clrcrl) [inline dynamic-data-masking-redshift/macros/apply_data_masking.sql]: this does not show up in the compiled SQL

File: dynamic-data-masking-redshift/macros/apply_data_masking.sql [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+-- what are the columns in our model?
+{% set model_cols = adapter.get_columns_in_relation(this) %}
+
+-- create schema if not exists {{ schema }};
+
+create view {{ schema }}.{{ this.identifier }} as (
+
+    select
+        {% for col in model_cols %}
+            {{ log(col.name, info=True) }}
```
Comment (clrcrl) [inline dynamic-data-masking-redshift/macros/apply_data_masking.sql]: this shows up on my command line (expected)

File: dynamic-data-masking-redshift/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# dynamic-data-masking
```
Comment (clrcrl) [inline dynamic-data-masking-redshift/README.md]: @dave-connors-3 — let's put a link to the Discourse article here when it's published :)

Comment (dave-connors-3) [reply dynamic-data-masking-redshift/README.md]: agreed!

**Pull request merged**